uitools/README.md

```diff
@@ -112,18 +112,13 @@ get added to QtCreator when you install the ArcGIS Runime SDK for Qt.
 
 - In QtCreator choose `File/New file or project/ArcGIS/ArcGIS Maps 200.4.0 Qt Quick C++ App`
 - choose settings to match the platform etc. you are building for
-- in the `CMakeLists.txt` file of your new app, add this line to get access to the `setup_toolkit` function:
-
-```CMake
-include(path/to/toolkitcpp.cmake)
-```
-- in the `CmakeLists.txt` file of your new app, call `setup_toolkit` after you've defined your target and pass in your target name and the path to the toolkit
-```CMake
-qt_add_executable(target_name ${SOURCE_FILES})
-...
-setup_toolkit(target_name path_to_toolkit)
-```
-
+- In the `CMakeLists.txt` file of your new application, you need to make two changes:
+  - Uncomment the `include` line below, and replace `path_to_toolkitcpp.cmake` with the actual path to your `toolkitcpp.cmake` file.
+  - Uncomment the `setup_toolkit` line below, and replace `target_name` with the name of your target, and `path_to_toolkit` with the path to your toolkit directory.
```

Comment:
```suggestion
  - Uncomment the `setup_toolkit` line below, and replace `target_name` with the name of your target, and `/path/to/` with the path to your toolkit directory.
```

Reply:
I see you put `/path/to/` but i went with `path/to/toolkit`. If you think thats confusing because we also reference `path/to/toolkitcpp/cmake`, let me know and I'll change it.